app/views/layouts/mailer.html.erb

```diff
@@ -1027,11 +1027,13 @@ h1 {
               <div class="column__padding--inline">
                 <div class="email-footer__address--inline">
                   <div>
+                    <p>P.S. You are receiving this email as a registered member of Ruby Australia. Australian legislation requires that we inform all members of Ruby Australia about our AGM and SGMs. If you no longer wish to receive these notifications, you can resign your membership from Ruby Australia <a href="https://ruby.org.au/my/details">here<a/></p>
```

Comment:
The markup for the link is closed incorrectly. Should be:

`<a href="https://ruby.org.au/my/details">here</a>`

Reply:
Sorry about the typo, my hands retired earlier today..  good catch!!

Both are fixed on the next commit.